src/pc/Win/include/win_time.h

```diff
@@ -2,6 +2,10 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#ifdef __GNUC__
```

Comment:
All of the CI build errors seem to be time related. Possibly the issue is build order dependent and emerged due to the parallel build, and adding a guard around the inclusion of win_time.h includes will eliminate the link time errors for "multiple definition of ..."